test/resources/transform_test/log_abi1.json

```diff
@@ -0,0 +1,72 @@
+[
+  {
+    "constant": true,
+    "inputs": [
+      {
+        "name": "addr",
+        "type": "address"
+      },
+      {
+        "name": "i8",
+        "type": "int8"
+      },
+      {
+        "name": "i16",
+        "type": "int16"
+      },
+      {
+        "name": "i32",
+        "type": "int32"
+      },
+      {
+        "name": "i64",
+        "type": "int64"
+      },
+      {
+        "name": "ui8",
+        "type": "uint8"
+      },
+      {
+        "name": "ui16",
+        "type": "uint16"
+      },
+      {
+        "name": "ui32",
+        "type": "uint32"
+      },
+      {
+        "name": "ui64",
+        "type": "uint64"
+      },
+      {
+        "name": "bool",
+        "type": "bool"
+      },
+      {
+        "name": "f128x20",
+        "type": "fixed128x20"
+      },
+      {
+        "name": "uf128x20",
+        "type": "ufixed128x20"
+      },
+      {
+        "name": "bytes",
+        "type": "bytes"
+      },
+      {
+        "name": "str",
+        "type": "string"
+      },
+      {
+        "name": "int8_array",
+        "type": "int8[6]"
```

Comment:
Suggest to add more cases of "array of ABI types", e.g. bytesM[N], bool[N], address[N],etc